test/e2e/monitoring/alerts_test.go

```diff
@@ -139,5 +147,82 @@ var _ = Context("Prometheus Alerts", func() {
 				ScaleDeployment(components.Name, components.Namespace, 1)
 			})
 		})
+
+		Context("and there are duplicate MACs", func() {
+			var virtClient kubecli.KubevirtClient
+			var err error
+
+			BeforeEach(func() {
+				By("starting virtClient")
+				virtClient, err = kubecli.GetKubevirtClientFromFlags("", os.Getenv("KUBECONFIG"))
+				Expect(err).ToNot(HaveOccurred())
+
+				By("creating test namespace")
```

Comment:
"creating test namespace that is not managed by kubemacpool (opted-out)"

Reply:
changed